api/services/managedInstance/aws/paths/managedIDCosts.yaml

```diff
@@ -9,7 +9,14 @@ get:
     - "Managed Instance AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../parameters/mi-id.yaml"
```

Comment:
@vlad666spot why changing? lets try use the commons. same for 'fromDate', 'toDate'.

Reply:
fixed